apps/web/src/pages/brand/tabs/v2/BrandingForm.tsx

```diff
@@ -0,0 +1,232 @@
+
+  return (
+    <SettingsPageContainer title="Branding">
+      <LoadingOverlay visible={!organization}>
+        <form noValidate onSubmit={handleSubmit(saveBrandsForm)}>
+          <Stack gap={32}>
+            <Stack gap={10}>
+              <Text weight="bold">Brand logo</Text>
+              <Controller
+                render={({ field }) => (
+                  /**
+                   * group className is used to target the dropzone overlay styles
+                   * REF: https://panda-css.com/docs/concepts/conditional-styles#group-selectors
+                   */
+                  <div className={cx('group', dropzoneWrapperStyles)}>
+                    {field.value && (
+                      <div className={dropzoneOverlayStyles}>
+                        <div>
+                          <ActionButton onClick={() => dropzoneRef.current?.()} Icon={IconOutlineFileUpload} />
+                          <Text color={colors.B70}>Update</Text>
+                        </div>
+
+                        <div>
+                          <ActionButton onClick={removeFile} Icon={IconDelete} />
+                          <Text color={colors.B70}>Remove</Text>
+                        </div>
+                      </div>
+                    )}
+                    <Dropzone
+                      styles={{
+                        root: {
+                          background: 'none',
+                          border: 'none',
+                        },
+                      }}
+                      openRef={dropzoneRef}
+                      accept={Object.keys(MIME_TYPE_TO_FILE_EXTENSION)}
+                      multiple={false}
+                      onDrop={handleUpload}
+                      data-test-id="upload-image-button"
+                    >
+                      <div
+                        className={css({
+                          minHeight: '6.25rem',
+                          minWidth: '6.25rem',
+                          display: 'flex',
+                        })}
+                      >
+                        {!field.value ? (
+                          <Stack gap="4px" direction="column" align="center" justify="center" flexGrow={1}>
+                            <ActionButton onClick={() => dropzoneRef.current?.()} Icon={IconOutlineFileUpload} />
+                            <Text color={colors.B70}>Upload</Text>
+                          </Stack>
+                        ) : (
+                          <img
+                            data-test-id="logo-image-wrapper"
+                            src={field.value}
+                            className={css({
+                              minHeight: '6.25rem',
+                              minWidth: '6.25rem',
+                              objectFit: 'contain',
+                            })}
+                            alt="avatar"
+                          />
+                        )}
+                      </div>
+                    </Dropzone>
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** This would be a good boundary for a standalone component -- please separate out‏